web/src/stream_settings_ui.js

```diff
@@ -151,7 +151,7 @@ export function update_stream_privacy(slim_sub, values) {
 
     // Update UI elements
     update_left_panel_row(sub);
-    if (narrow_state.stream_sub().stream_id === sub.stream_id) {
+    if (narrow_state.stream_sub()?.stream_id === sub.stream_id) {
```

Comment:
We do this at couple of other places as well. Or should we do `narrow_state.stream_sub() !== undefined` check?

Reply:
I think I prefer this version.